test-network-function/operator/suite.go

```diff
@@ -143,79 +132,50 @@ func getConfig() ([]configsections.CertifiedOperatorRequestInfo, []configsection
 }
 
 func itRunsTestsOnOperator() {
-	testID := identifiers.XformToGinkgoItIdentifier(identifiers.TestOperatorIsCertifiedIdentifier)
-	ginkgo.It(testID, func() {
```

Comment:
Was working on the same code this it statement was shadowing the one below, line 208, now looks ok without the duplicate test case